fe/fe-core/src/main/java/org/apache/doris/service/FrontendServiceImpl.java

```diff
@@ -822,7 +822,7 @@ private boolean loadTxnCommitImpl(TLoadTxnCommitRequest request) throws UserExce
             throw new UserException("unknown database, database=" + dbName);
         }
 
-        long timeoutMs = request.isSetThriftRpcTimeoutMs() ? request.getThriftRpcTimeoutMs() : 5000;
+        long timeoutMs = request.isSetThriftRpcTimeoutMs() ? request.getThriftRpcTimeoutMs() / 2 : 5000;
```

Comment:
could you please explain why divide by 2?

Reply:
This is an experience value, here we just ensure that the rpc_timeout value is larger enough than  timeoutMs  to keep timeout setting consistent with before, to reduce the "call frontend service failed error" occur times. But I think there is still some improvement we can do that we should compute the left time that  timeoutMillis minus commit time cost to improve the  Probability of publish timeout instead of "call frontend service failed" when commit or publish version performance is poor